lib/matplotlib/widgets.py

```diff
@@ -2511,13 +2531,13 @@ def onselect(eclick: MouseEvent, erelease: MouseEvent)
         self.spancoords = spancoords
         self._drawtype = drawtype
 
-        self.maxdist = maxdist
+        self.handle_grab_distance = handle_grab_distance
```

Comment:
This needs a deprecation accessor on the old `maxdist` attribute.

Reply:
Yes, indeed, done!